1.generate-profiles/profiling_config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+core:
+  step: generate-profiles
+  pipeline: Pooled Cell Painting
+  project: 2018_11_20_Periscope_Calico
+  batch: 20190919_6W_CP074A
+  project_dir: /Users/gway/work/projects/
+  site_dir: ../0.preprocess-sites/data/
+  output_single_cell_dir: single_cell/
+  output_profile_dir: profiles/
+  categorize_cell_quality: simple
+  compression: gzip
+  float_format: "%.5g"
+  compartments:
+    - Cells
+    - Nuclei
+    - Cytoplasm
+  parent_cols:
+    cells:
+      - Parent_Nuclei
+    cytoplasm:
+      - Parent_Nuclei
+      - Parent_Cells
+    spots:
+      - Parent_Cells
+  id_cols:
+    - ImageNumber
+    - ObjectNumber
+  control_barcodes:
+    - NT
+  trash_files:
+    - .DS_Store
+---
+single_cell:
+  perform: true
+  prefilter_features: true
+  filter_cells:
```

Comment:
What you have as `filter_cells` here is `cell_filter` in the site_processing_config.yaml. We should pick one? This is passed to 1.generate-profiles/0.merge-single-cells.py as `filter_cell_quality` which I like even better because it's more descriptive. Can we switch to `filter_cell_quality` throughout?

Reply:
yep, I agree this is more descriptive